### ISSUE TITLE
SP / Geonetwork - Multiple HTTP 400 errors with editor

### ISSUE THREAD

**Issue by jusabatier** [georchestra/georchestra#1948]
When trying to create MDs on GeoNetwork, I notice that I had often http 400 errors.

The concrete use case I tested with is : 

Create a contact to import in MDs
Import the contact with the search contect function as a "Point of contact".

This show the error message :  

> Une erreur est survenue lors de l'appel du service
> Certaines erreurs peuvent être corrigées dans la vue XML.

When looking in the browser console, it's due to an http 400 with no response.

In the SP logs, I have : 

> security [WARN] Error occured. statuscode: 400, reason:

I tried to access directly to GeoNetwork (http://<geonetwork_serv_ip>:8080/geonetwork and it work fine.

After I tried to access directly to the SP and it fail.

Here is the curl copy of the request : 

https://pastebin.com/kDt3SMBS

And something more readable, the POST data : 

https://pastebin.com/P1u4j0Kg

I think the SP have problems with POST datas parsing or length.

**Comment by fvanderbiest** [comment]
Which branch are you testing ? Thanks

**Comment by jusabatier** [comment]
The 17.12

**Comment by jusabatier** [comment]
I played a little with different confs for Tomcat's connector, and the problem seems to be solved by adding : 

maxHttpHeaderSize="12288"

to the SecProxy's and Geonetwork's connectors.

It look like when using NGINX+SP+GeoNetwork the size of request header is more important than the default one in TC (4KB).

I tried to set it to 12KB, don't know if it is the best suited value, but it seems to solve the problem for now.
